php-practice_2.php

```diff
@@ -1,15 +1,99 @@
 <?php
 // Q1 tic-tac問題
+for ($i = 1; $i <= 100; $i++) {
+  if ($i % 4 == 0 && $i % 5 == 0) {
+    echo 'tic-tac' ."\n";
+  } else
+  if ($i % 5 == 0) {
+    echo 'tac' ."\n";
+  } else
+  if ($i % 4 == 0) {
+    echo 'tic'."\n";
+  } else 
+  echo $i."\n";
+
+}
```

Comment:
[nits]
インデントの修正お願いいたします！
```suggestion
for ($i = 1; $i <= 100; $i++) {
  if ($i % 4 == 0 && $i % 5 == 0) {
    echo 'tic-tac' ."\n";
  } else if ($i % 5 == 0) {
    echo 'tac' ."\n";
  } else if ($i % 4 == 0) {
    echo 'tic'."\n";
  } else 
  echo $i."\n";
}
```

Reply:
失礼いたしました。
以下のような修正の仕方でよろしいでしょうか？

  if ($i % 4 == 0 && $i % 5 == 0) {
    echo 'tic-tac' . "\n";

  } elseif ($i % 5 == 0) {
    echo 'tac' . "\n";

  } elseif ($i % 4 == 0) {
    echo 'tic'. "\n";

  } else 
  echo $i . "\n";
}